pkg/types/installconfig.go

```diff
@@ -101,20 +102,21 @@ func (p *Platform) Name() string {
 
 // Networking defines the pod network provider in the cluster.
 type Networking struct {
-	Type        NetworkType `json:"type"`
+	// Type is the network type to install
+	Type netopv1.NetworkType `json:"type"`
+
+	// ServiceCIDR is the ip block from which to assign service IPs
 	ServiceCIDR ipnet.IPNet `json:"serviceCIDR"`
-	PodCIDR     ipnet.IPNet `json:"podCIDR"`
-}
 
-// NetworkType defines the pod network provider in the cluster.
-type NetworkType string
+	// ClusterNetworks is the IP address space from which to assign pod IPs.
+	ClusterNetworks []netopv1.ClusterNetwork `json:"clusterNetworks,omitempty"`
 
-const (
-	// NetworkTypeOpenshiftSDN is used to install with SDN.
-	NetworkTypeOpenshiftSDN NetworkType = "openshift-sdn"
-	// NetworkTypeOpenshiftOVN is used to install with OVN.
-	NetworkTypeOpenshiftOVN NetworkType = "openshift-ovn"
-)
+	// PodCIDR is deprecated (and badly named; it should have always
+	// been called ClusterCIDR. If no ClusterNetworks are specified,
+	// we will fall back to the PodCIDR
+	// TODO(cdc) remove this.
+	PodCIDR *ipnet.IPNet `json:"podCIDR,omitempty"`
```

Comment:
can we open an issue tracking this. JIRA or github either one is fine.

Reply:
SDN-235